src/main/java/seedu/address/model/person/Priority.java

```diff
@@ -17,7 +17,7 @@ public class Priority {
     // Error message for invalid priority values.
     public static final String MESSAGE_CONSTRAINTS = "Priority should only be 'NONE', 'LOW', 'MEDIUM', or 'HIGH'.";
 
-    private final String priority;
+    public final String priority;
```

Comment:
I would prefer making this private, then use IntelliJ to generate (alt+insert) getter and setter. Saves lots of headache if you want to ensure some things (like format it or check validity when setting the priority so that no one else in our group sets an invalid priority)

Reply:
Had the same idea, but went through with following the standard convention of other classes under model/person where all the values are public. Had I set it to private my getters would have to be different name as we would then need 2 getters to get the actual priority in String format (i.e first getter to get Priority class object from Person object, then second getter to get the String value) making the test cases less comprehensible, as such decide to go with the flow to ensure readability as well as standardizations throughout test cases.

Reply:
Also in Person class , the Priority variable is Private final already, so it is not that detrimental to set priority's own variable to be public to reduce having 2 getters with similar names.